plugins/discourse-lazy-videos/assets/stylesheets/lazy-videos.scss

```diff
@@ -0,0 +1,138 @@
+.lazy-video-container {
+  z-index: z("base");
+  position: relative;
+  display: block;
+  height: 0;
+  padding: 0 0 56.25% 0;
+  background-color: #000;
+  margin-bottom: 12px;
+
+  .video-thumbnail {
+    cursor: pointer;
+    overflow: hidden;
+    height: 0;
+    padding: 0 0 56.25% 0;
+
+    img {
+      object-fit: cover;
+      width: 100%;
+      pointer-events: none;
+    }
+
+    &:hover,
+    &:focus {
+      .icon {
+        transform: translate(-50%, -50%) scale(1.1);
+      }
+    }
+
+    &:focus {
+      outline: 5px auto Highlight;
+      outline: 5px auto -webkit-focus-ring-color;
+    }
+
+    &:active {
+      outline: 0px;
+    }
+  }
+
+  .title-container {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    top: 0;
+    z-index: 935;
```

Comment:
Nitpick, but I'm not sure we need the z-index here. Seems to work fine without it.